### PR TITLE
Adds a way to build a distribution and deploy it to a SUL cdn

### PR DESCRIPTION
**Pull request by mejackreed**
comments please from @jkeck @cbeer and @eefahy

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -9,6 +9,23 @@ task default: :ci
...
+  FileUtils.mkdir_p(dist_dir)
+
+  # Copy the fonts over to the dist directory
+  FileUtils.cp_r(asset_dir + '/fonts/.', dist_dir)
+
+  # Create sul-icons.css
+  `sass -C --sourcemap=none #{asset_dir + '/stylesheets/_sul-icons.scss'} #{dist_dir + '/sul-icons.css'}`
+
+  # Create a minified version of sul-icons.css
+  `sass -C --sourcemap=none --style=compressed #{asset_dir + '/stylesheets/_sul-icons.scss'} #{dist_dir + '/sul-icons.min.css'}`
```
Comment (jkeck) [inline Rakefile]: Is the `sass` executable provided by sass-rails?  If so, I wonder if we need to do anything to make sure this is executed in the correct context.  I would be totally for waiting until it presents an issue to solve it (given that I don't know if it'll even be a problem)

Comment (mejackreed) [inline Rakefile]: Yeah... I think so.. https://github.com/rails/sass-rails/blob/master/sass-rails.gemspec#L17

I tried doing it differently straight through Ruby, but I was basically recreating this... seemed to be the simplest option.

Comment (jkeck) [inline Rakefile]: Totally understand, we may want to execute this via `bundle exec` but I'm not sure that's even a proper thing to do in this context.

Comment (mejackreed) [inline Rakefile]: :+1: good idea.. thanks.

**Comment by jkeck** [comment]
@mejackreed should we :shipit: or wait for additional feedback?

**Comment by mejackreed** [comment]
I was going to wait until we got apache up and going to see if everything actually worked.

**Comment by jkeck** [comment]
Ah, sorry, thought that had already happened.

**Comment by eefahy** [comment]
Was working on pipeline today. I can do the apache stuff on Monday.

**Comment by jkeck** [comment]
:+1: thanks @eefahy:exclamation:

**Comment by mejackreed** [comment]
@jkeck :boat: this!

**Pull request merged**